### PR TITLE
Completed task 6 - Get Article Comments

### PR DESCRIPTION
**Pull request by BlurryQ**

### CODE REVIEW

File: models/articles.models.js [hunk 1]
```diff
@@ -30,3 +30,21 @@ exports.selectArticleByID = (article_id) => {
...
+exports.selectArticleComments = (article_id) => {
+  return db
+    .query(
+      `select * from comments
+      where article_id = $1
+      order by created_at desc`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0)
```
Comment (nc-hannah-burks) [inline models/articles.models.js:43]: You also need to check the reason for the query returning empty rows, as this could be one of two reasons:
article exists but has no comments.
OR
article does not exist.
In order to determine the reason why, you will need to check whether the article exists first.

File: __tests__/integrations.test.js [hunk 2]
```diff
@@ -152,4 +152,55 @@ describe("Endpoint testing for NC News", () => {
...
+    });
+    it("400: respond msg 'bad request' when invalid datatype is entered as the article ID", () => {
+      return request(app)
+        .get("/api/articles/one/comments")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("bad request");
+        });
+    });
+    it("404: respond msg 'not found' when a valid article ID datatype is entered, but article does not exist or contain any comments", () => {
```
Comment (nc-hannah-burks) [inline __tests__/integrations.test.js:197]: Ideally we also want a test for 	Status 200, valid ID, but has no comments responds with an empty array of comments
